crates/starfish/core/src/base_committer.rs

```diff
@@ -26,8 +26,7 @@ mod base_committer_tests;
 mod base_committer_declarative_tests;
 
 pub(crate) struct BaseCommitterOptions {
-    /// TODO: Re-evaluate if we want this to be configurable after running
-    /// experiments. The length of a wave (minimum 3)
+    /// The length of a wave is set to a hardcoded constant 3
```

Comment:
Why do we have it in the options if there is no possibility to change this and the value is sourced from the constant anyway? IMO we should use the constant value instead.